funnel/models/space.py

```diff
@@ -110,23 +120,29 @@ def proposals_all(self):
 
     @property
     def proposals_by_status(self):
-        from .proposal import Proposal, PROPOSALSTATUS
+        from .proposal import Proposal
         if self.subspaces:
             basequery = Proposal.query.filter(Proposal.proposal_space_id.in_([self.id] + [s.id for s in self.subspaces]))
         else:
             basequery = Proposal.query.filter_by(proposal_space=self)
-        return dict((status, basequery.filter_by(status=status).order_by(db.desc('created_at')).all()) for (status, title) in PROPOSALSTATUS.items() if status != PROPOSALSTATUS.DRAFT)
+        all_proposals = basequery.filter(Proposal.state.NOT_DRAFT).order_by(db.desc('created_at')).all()
```

Comment:
What happens if you use `basequery.filter(not Proposal.state.DRAFT)`? We haven't tested this in Coaster. We should.

Reply:
This doesn't work. it becomes `WHERE true` as `not Proposal.state.DRAFT` becomes `True`